packages/safe-tools-client/app/components/deposit-modal/index.gts

```diff
@@ -0,0 +1,128 @@
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { later } from '@ember/runloop';
+
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+import copyToClipboard from '@cardstack/boxel/helpers/copy-to-clipboard';
+import cssVar from '@cardstack/boxel/helpers/css-var';
+import BoxelModal from '@cardstack/boxel/components/boxel/modal';
+import BoxelActionContainer from '@cardstack/boxel/components/boxel/action-container';
+import BoxelInputGroup from '@cardstack/boxel/components/boxel/input-group';
+
+import './index.css';
+
+interface MonthlyTokensToCover {
+  tokens: Array<{ 
+    symbol: string; 
+    amountToCover: number; 
+  }>;
+  hasEnoughBalance: boolean;
+}
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    isOpen: boolean;
+    onClose: () => void;
+    safeAddress: string;
+    networkName: string;
+    tokensToCover?: {
+      nextMonth: MonthlyTokensToCover;
+      nextSixMonths: MonthlyTokensToCover;
+    };
+  };
+}
+
+
+export default class DepositModal extends Component<Signature> {
+  @tracked isShowingCopiedConfirmation = false;
+
+  @action flashCopiedConfirmation() {
+    this.isShowingCopiedConfirmation = true;
+    later(() => {
+      this.isShowingCopiedConfirmation = false;
+    } , 1000)
+  }
+
+  <template>
+    <BoxelModal @size="small" @isOpen={{@isOpen}} @onClose={{@onClose}}>
+      <BoxelActionContainer as |Section ActionChin|>
+        <Section @title="Deposit Instructions" class="deposit-modal__section">
+          <p>
+            it is the user's responsibility to ensure that sufficient funds are
```

Comment:
```suggestion
            It is the user’s responsibility to ensure that sufficient funds are
```

The design is inconsistent re the typographer apostrophe but we might as well be consistent and use it 🤓 

<img width="1032" alt="Schedule Payment - Deposit Modal - MVP - Zeplin 2022-10-24 09-27-59" src="https://user-images.githubusercontent.com/43280/197550639-1ea7a802-321b-4724-94bc-171b1844facb.png">

Reply:
wow, good eye, I'll be more attentive to use `’`